core/src/main/java/psiprobe/tools/AccessorFactory.java

```diff
@@ -43,7 +43,13 @@ public static Accessor getInstance() {
   private static Accessor getReflective() {
     try {
       return new ReflectiveAccessor();
-    } catch (Exception ex) {
+    } catch (ClassNotFoundException e) {
```

Comment:
CodePro and/or Findbugs will flag that as an issue due to lack of logging.
With the other logging work stream going we can ignore for now, but once
logging is finalized, let's come back and add logging (even if debug) in
places like this and others.

On Sun, Feb 7, 2016, 11:16 PM Jeremy Landis <notifications@github.com>
wrote:

> In core/src/main/java/psiprobe/tools/AccessorFactory.java
> <https://github.com/psi-probe/psi-probe/pull/655#discussion_r52132548>:
>
> > @@ -43,7 +43,13 @@ public static Accessor getInstance() {
> >    private static Accessor getReflective() {
> >      try {
> >        return new ReflectiveAccessor();
> > -    } catch (Exception ex) {
> > +    } catch (ClassNotFoundException e) {
>
> At the moment, we are still compiling to java 6. Do you know if
> multi-catch is something we can code and compile towards 6? We haven't
> bumped up fully to 7 yet.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/psi-probe/psi-probe/pull/655/files#r52132548>.
>